pkg/lang/javascript/parser.go

```diff
@@ -12,8 +12,10 @@ import (
 
 var multilineCommentMarginRegexp = regexp.MustCompile(`(?m)^\s*[*]*[ \t]*`) // we need to use [ \t] instead of \s, because \s includes newlines in (?m) mode.
 
+const js = core.LanguageId("javascript")
```

Comment:
I moved this into the struct a while back and just use `Language.ID` (since it's exported and IMO clearer what it is).